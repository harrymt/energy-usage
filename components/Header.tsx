import React from 'react'

export const Header = ({}) => <div>Header</div>

export default Header
