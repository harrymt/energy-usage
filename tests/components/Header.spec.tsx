import React from 'react'
import renderer from 'react-test-renderer'
import Header from '../../components/Heading'

describe('Header', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Header />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
