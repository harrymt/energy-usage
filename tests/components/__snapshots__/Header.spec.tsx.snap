// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<h1>
  Energy Usage
</h1>
`;
